src/components/UserInfo.jsx

```diff
@@ -0,0 +1,182 @@
+    }
+  },
+  link: {
+    display: 'flex',
+    alignItems: 'center',
+    '& > a': {
+      margin: theme.spacing(0, 0, 0, 1)
+    }
+  },
+  social: {
+    '& > div': {
+      marginBottom: theme.spacing(1)
+    }
+  },
+  bio: {
+    margin: theme.spacing(3, 0)
+  },
+  interested: {
+    '& > a': {
+      display: 'flex',
+      alignItems: 'center',
+      margin: theme.spacing(0, 0, 1),
+      '&:hover': {
+        textDecoration: 'underline'
+      },
+      '& > span': {
+        width: '24px',
+        height: '24px',
+        margin: theme.spacing(0, 1, 0, 0)
+      }
+    }
+  },
+  ellipsis: {
+    overflow: 'hidden',
+    whiteSpace: 'nowrap',
+    textOverflow: 'ellipsis'
+  }
+});
+
+function UserInfo({
+  classes,
+  user: {
+    id,
+    firstName,
+    lastName,
+    imgSrc,
+    bio,
+    link,
+    username,
+    followingCount,
+    followersCount,
+    interested
+  }
+}) {
+  const avatarSection = (
+    <section>
+      <Paper className={classes.avatar}>
+        <Avatar alt={username} src={imgSrc} variant="square" />
+      </Paper>
+      <Typography variant="h5">
+        {firstName} {lastName}
+      </Typography>
+    </section>
+  );
+
+  const linkSection = Boolean(link) && (
+    <section className={classes.link}>
+      <LinkIcon />
+      <MUILink
+        className={classes.ellipsis}
+        href={link}
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        {link}
+      </MUILink>
+    </section>
+  );
+
+  const socialSection = (
+    <section className={classes.social}>
+      <Grid container spacing={2}>
+        <Grid item xs={6}>
+          <MUILink href="">
+            <Box fontWeight="fontWeightBold" component="span">
+              {followersCount}&nbsp;
+            </Box>
+            {followersCount === 1 ? 'Follower' : 'Followers'}
+          </MUILink>
+        </Grid>
+        <Grid item xs={6}>
+          <MUILink href="">
+            <Box fontWeight="fontWeightBold" component="span">
+              {followingCount}&nbsp;
+            </Box>
+            Following
+          </MUILink>
+        </Grid>
+      </Grid>
+      <Button fullWidth>Follow</Button>
+    </section>
+  );
+
+  const bioSection = (
+    <section className={classes.bio}>
+      <Typography>{bio}</Typography>
+    </section>
+  );
+
+  const interestedSection = Boolean(interested.length) && (
+    <section className={classes.interested}>
+      <Typography gutterBottom>Interested in projects</Typography>
+      {interested.map((project, index) => (
+        <Link key={index} to={`/project/${project.id}`}>
+          <Avatar alt={project.title} component="span" src={project.logo} />
+          <Typography className={classes.ellipsis}>{project.title}</Typography>
+        </Link>
+      ))}
+    </section>
+  );
+
+  return (
+    <div className={classes.root}>
+      {avatarSection}
+      {linkSection}
+      {socialSection}
+      {bioSection}
+      {interestedSection}
+    </div>
```

Comment:
The UserInfo component looks great on desktop, but the avatar becomes a bit large on the mobile breakpoint. I'm wondering if having a profile card for mobile with the avatar on the left and the profile info on the right would be a clean solution? The project cards would go below that, as you currently have it.

![image](https://user-images.githubusercontent.com/47229249/82949935-4cf12800-9f59-11ea-8b4b-134efcc9bb40.png)

Reply:
Hahaha yes, I actually haven't looked at it on mobile at all. I'll do the mobile for the profile page in another PR.